crates/telio-dns/tests/integration_tests.rs

```diff
@@ -53,16 +41,19 @@ impl fmt::Display for WireguardConfig {
         write!(
             f,
             r#"[Interface] # Regular peer
-                PrivateKey = {}
+                PrivateKey = {:?}
```

Comment:
Why did it become debug here, display should show base64, debug concat's

Reply:
Done